noxfile.py

```diff
@@ -109,7 +109,7 @@ def integration(session: nox.Session):
         "--source",
         "antsibull_changelog",
         "-m",
-        "antsibull_changelog.cli",
+        "antsibull_changelog.__main__",
```

Comment:
```suggestion
        "antsibull_changelog",
```

should be sufficient.

Reply:
True, but this is more explicit (and easier to find the connection between the integration tests and `src/antsibull_changelog/__main__.py`).

Reply:
I guess, it just seems redundant to me, and I've never seen -m used like this. The [`python -m` docs](https://docs.python.org/3/using/cmdline.html#cmdoption-m) say:

> Package names (including namespace packages) are also permitted. When a package name is supplied instead of a normal module, the interpreter will execute `<pkg>.__main__` as the main module.

Reply:
Ok, then let's use it.